themes/gfsc/layouts/partials/projectindex.html

```diff
@@ -3,7 +3,7 @@
       <a href="{{ .URL }}" class="work__project__title">
          <h2 class="work__project__title">{{ .Title }}</h2>
       </a>
-      <p class="work__project__summary">{{ .Params.summary }}</p>
+      <p class="work__project__summary">{{ .Description }}</p>
```

Comment:
Does it need to be Params.description?

Reply:
no - title and description don't need it as i think they're seen as have to haves, everything else does tho